CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Default Contributions Catalog port to 5000 and made it configurable. [#788](https://github.com/rokwire/rokwire-building-blocks-api/issues/788)
 
 ### Fixed
+- Successful POST of contribution redirect to contribution details page. [#777](https://github.com/rokwire/rokwire-building-blocks-api/issues/777)
```

Comment:
Please move this entry under the Unreleased header. Within a section, according to the CHANGELOG format, we need to put the most recent entry at the bottom. Thanks.

Reply:
Also, the second task linked in this PR (#774) needs to be mentioned in the CHANGELOG. Thanks.

Reply:
Changed and added